smart_arg.py

```diff
@@ -108,6 +110,7 @@ class MyArg(NamedTuple):
                       f"unsupported python version {sys.version_info} < 3.6. It might work with 'pip install typing_inspect'.")
         raise
 
+frozenlist = tuple  # tuple to simulate an immutable list. [https://www.python.org/dev/peps/pep-0603/#rationale]
```

Comment:
why you give a name of `frozenlist` instead of directly telling user to use `tuple`?

Reply:
`tuple` works too, but `frozelist` is more expressive and comparable to `frozenset`.